### DISCUSSION TITLE
Ability to ignore the step result

### DISCUSSION THREAD

**Discussion by porkumaran4** [testsigmahq/testsigma#12] [Feature Ideas]
There will be use cases like we need to ignore less priority steps in the test report, so the result of those cases need not to be considered for the test case result.

**Comment by rukmangada** [comment]
Looks like a valid use case we will add to the list and let you know when we prioritize this

**Comment by PratheepV** [comment]
@porkumaran4 We build the feature and it's currently in the QA phase. Will keep you posted.

Will be going on 1.2.0 release
